### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by TheaDene** [benmaier/qsuite#14]
When I use `qsuite stat` and the last calculation is still running, sometimes I get the following error:
```
Traceback (most recent call last):
  File "/anaconda/bin/qsuite", line 11, in <module>
    load_entry_point('qsuite==0.4.4', 'console_scripts', 'qsuite')()
  File "/anaconda/lib/python2.7/site-packages/qsuite-0.4.4-py2.7.egg/qsuite/qsuite_binary.py", line 463, in main
    print_status(cf,ssh)
  File "/anaconda/lib/python2.7/site-packages/qsuite-0.4.4-py2.7.egg/qsuite/printparams.py", line 83, in print_status
    prog.append( [ old_id ] + progresses[old_id])
IndexError: list index out of range
```

**Comment by benmaier** [comment]
```
ssh> for i in `seq 0 0`; do cat /home/bfmaier/fixpkt_ana/fixpkt_ana_10.0_NMEAS_1_ONLYSAVETIME_False/output/progress_$i; done;
Traceback (most recent call last):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/bin/qsuite", line 11, in <module>
    load_entry_point('qsuite', 'console_scripts', 'qsuite')()
  File "/Users/bfmaier/Seafile/forschung/independent-python-packages/qsuite/qsuite/qsuite_binary.py", line 464, in main
    print_status(cf,ssh)
  File "/Users/bfmaier/Seafile/forschung/independent-python-packages/qsuite/qsuite/printparams.py", line 83, in print_status
    prog.append( [ old_id ] + progresses[old_id])
IndexError: list index out of range
```

**Comment by benmaier** [comment]
potentially fixed together with #21
